mu/modes/circuitpython.py

```diff
@@ -24,46 +24,24 @@
 from mu.interface.panes import CHARTS
 
 
-class AdafruitMode(MicroPythonMode):
+class CircuitPythonMode(MicroPythonMode):
     """
-    Represents the functionality required by the Adafruit mode.
+    Represents the functionality required by the CircuitPython mode.
     """
 
-    name = _('Adafruit CircuitPython')
-    description = _("Use CircuitPython on Adafruit's line of boards.")
-    icon = 'adafruit'
-    save_timeout = 0  #: Don't autosave on Adafruit boards. Casues a restart.
-    connected = True  #: is the Adafruit board connected.
+    name = _('CircuitPython')
+    description = _("Boards running Circuit Python.")
+    icon = 'circuitpython'
+    save_timeout = 0  #: Noautosave on CP boards. Will restart.
```

Comment:
Is `Noautosave` a term? or just a missing space?

Reply:
```suggestion
    save_timeout = 0  #: No auto-save on CP boards. Will restart.
```

Reply:
Just a typo - spacing issue.